cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerHttpHandlerInternal.java

```diff
@@ -104,23 +97,25 @@ public TaskWorkerHttpHandlerInternal(CConfiguration cConf, Consumer<String> stop
     this.runnableTaskLauncher = new RunnableTaskLauncher(cConf);
     this.metricsCollectionService = metricsCollectionService;
     this.metadataServiceEndpoint = cConf.get(Constants.TaskWorker.METADATA_SERVICE_END_POINT);
-    this.stopper = (terminate, taskDetails) -> {
-      emitMetrics(taskDetails);
+    this.stopper = (succeeded, taskDetails) -> {
```

Comment:
There are 2 consumers here with the same variable name `stopper`. One is for TaskWorkerService and other for TaskWorkerHttpHandlerInternal. Can we name them appropriately?

Reply:
yeah.. the original code was not naming it very nicely. will update

Reply:
Updated the field to `taskCompletionConsumer`